src/types/TermControlUiaProvider.cpp

```diff
@@ -66,6 +64,11 @@ IFACEMETHODIMP TermControlUiaProvider::get_BoundingRectangle(_Out_ UiaRect* pRec
     return S_OK;
 }
 
+IFACEMETHODIMP TermControlUiaProvider::get_HostRawElementProvider(_COM_Outptr_result_maybenull_ IRawElementProviderSimple** ppProvider) noexcept
+{
+    return _controlInfo->GetHostUiaProvider(ppProvider);

```

Comment:
since this one is noexcept, but the ones under it aren't (?), you may want to wrap this in a function-level try;

```
try
{
...
}
CATCH_RETURN();
```

that'll convert all exceptions into HRESULTs at the COM boundary